_sections/100-encryption.md

```diff
@@ -0,0 +1,109 @@
+    <encryptedXmlFile>submission.xml.enc</encryptedXmlFile>
+    <media>
+        <file>myimage.jpg.enc</file>
+        <file>myaudio.mp3</file>
+    </media>
+    <base64EncryptedElementSignature>OU7rbZl0uFy7xv/HnSl1juVrdf2fQpzcfjwetgl+wseOx5yeD3NjoAg978GGclsy38mECEgTkMS1g8J1I/Xrn9uSQCRyaJXgPyFYPP+y24ka+vCNuNfg6SN1h8MYyUDdg7B7/M9oacMixbAtHo9qcesSBykJWJjFjBS7Nl/GnojRIc5ywLwnzKrdjjxeTjFw7kIG3LCt298WBHuj7azbi/DJYPp26Dbho47LlaRbQpi5Q4Oea71y1h7Wdbl4r7ILyRkTo86fvg6HUfWDLWSorgoFCqi1Af9qP2ziF+LLWQzDu3M8SCHX6uWdCRm/8GPaAyUpMAyfy2e8i7KPbMcVsQ==</base64EncryptedElementSignature>
+    <meta xmlns="http://openrosa.org/xforms">
+        <instanceID>uuid:5b9cf8d1-106f-4004-844f-c072d76762ed</instanceID>
+    </meta>
+</data>
+```
+
+The following elements are supported. Unless otherwise specified all attributes and elements should be in the `"http://opendatakit.org/submissions"` namespace.
+
+| `<element>`/attribute             | description
+|-----------------------------------|-------------
+|`<data>`                           | The required root element.
+|          `id`                     | Required on the root element with same value as the `id` attribute in the [Primary Instance](/#primary-instance) of the XForm.
+|         `encrypted`               | Required on the root element with the value "yes" indicating this is an encryption manifest.
+|         `version`                 | Required on the root element with same value as the `version` attribute in the [Primary Instance](/#primary-instance) of the XForm but only if it exist there.
+|`<base64EncryptedKey>`             | Required child of the root element with the value of the [encrypted encryption key](#key-encryption) used to encrypt the record.
+|`<encryptedXmlFile>`               | Required child of the root element with the value of the filename of the encrypted XML file.
+|`<media>`                          | Child of the root element, which is required if the record contains any additional media files.
+|`<file>`                           | Child of the media element, which is required for each media file belonging to the record. The order of multiple file elements is [significant](#content-encryption).
+|`<base64EncryptedElementSignature>`| Optional child of the root element that contains a [signature](#signature) that can be used to validate that the encrypted record does not appear to be tampered with.
+|`<meta>`                           | Required child of the root element in the `"http://openrosa.org/xforms"` namespace.
+|`<instanceID>`                     |  Required child of the meta element in the `"http://openrosa.org/xforms"` namespace containing the value of the identical element in the XML record.
+
+### Content Encryption
+
+The XML file and uploaded files are encrypted with the equivalent of the **AES/CFB/PKCS5Padding** algorithm as used in Java 9 with a [specified initialization vector algorithm](#initialization-vector). 
```

Comment:
Why Java 9 here? We are using Java 8

Reply:
Oh, I had no idea. Thanks. Will change to 8!